airbyte-webapp/src/components/ui/SideMenu/SideMenu.tsx

```diff
@@ -27,7 +27,7 @@ interface SideMenuProps {
 }
 
 const Content = styled.nav`
-  min-width: 147px;
+  min-width: 155px;
```

Comment:
The sidebar is ever-so-slightly too small for `Default Data Residency`. This CSS should really be more flexible, but just making it work for now!